examples/ssh/backup_ftps.js

```diff
@@ -0,0 +1,65 @@
+/**
+ * This driver is designed to verify if backup exist and have a certain size.
+ * Communication protocol is SSH.
+ * This driver create a dynamic monitoring variables 
+ * fileSize: to store file size.
+ * lastodif: to store last modification date.
+ */
+
+// Define SSH configuration object with port and timeout values
+var sshConfig = {
+    port: 22,
+    timeout: 5000
+};
+
+var fileSize, lastodif;
+var _var = D.device.createVariable;
```

Comment:
Please use D.createVariable directly, no need to re-define it like this as its now shorter anyways